tools/rush-update/src/rush.ts

```diff
@@ -22,10 +23,16 @@ export const getProjectFolders = once(() => getRushConfig()
   .then(({projects}) => projects.map((p) => p.projectFolder))
 );
 
-export async function updateRushShrinkwrapFile() {
+export async function updateRushShrinkwrapFile(): Promise<void> {
   const { installAndRun } = await importInstallRunScript();
   const statusCode = installAndRun('@microsoft/rush', await getRushVersion(), 'rush', ['update', '--full']);
   if (statusCode !== 0) {
     throw new Error(`'rush update' exited with code ${statusCode}`);
   }
 }
+
+export function generateRushChangeFiles(): void {
+  execSync('yes "" | rush change', {
```

Comment:
Can you add a comment that this answers the question `Describe changes, or ENTER if no changes:` with ENTER?

Reply:
adding